vars/buildExecute.groovy

```diff
@@ -100,5 +106,23 @@ void call(Map parameters = [:]) {
                     error "[${STEP_NAME}] buildTool not set and no dockerImage & dockerCommand provided."
                 }
         }
+        if (config.buildTool != 'docker' && config.cnbBuild) {
+            switch(config.buildTool){
+                case 'npm':
+                    script.commonPipelineEnvironment.setContainerProperty('buildpacks', ["gcr.io/paketo-buildpacks/nodejs"])
```

Comment:
Is it ok to hard code the build-pack here?

Reply:
Why would it be necessary to hard code the buildback? Either it is detected automatically or defined by the user.